sktime/utils/data.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+"""Utility functions for data handling."""
+
+__author__ = "James Large"
+
+import pandas as pd
+
+from sktime.datatypes._panel._convert import from_nested_to_3d_numpy
+from sktime.utils.validation.panel import check_X, check_X_y
+
+
+def check_and_clean_data(X, y=None, input_checks=True):
```

Comment:
much of this seems redundant with functionality now in `datatypes`.
Have a look at `convert_to`.

Also, if this is specific to neural networks, should it live in `networks`?

Reply:
I replaced this